doc/howtos/rdtraining/16_guidelines_pr.rst

```diff
@@ -0,0 +1,154 @@
+
+.. exercise:: Make it shine
+
+    Refactor your code to respect the coding guidelines. Don't forget to respect the module
+    structure, the variable names, the method name convention, the model attribute order and the
+    xml ids.
+
+Your first Pull Request (PR)
+============================
+
+**The following is intended to be used by Odoo staff members only. All repositories mentioned are
+not accessible by third-party.**
+
+Now that your code respects the Odoo code guidelines and is shiny, let's share it with others.
+
+
+Configure development repository
+--------------------------------
+
+We will start creating a git development environment for the 'custom' folder, as we did it in the
+beginning for 'odoo' and 'enterprise'. The PR will target the `odoo/technical-training-sandbox` repository.
+
+To do this the first step is to
+`fork this repository in GitHub <https://guides.github.com/activities/forking/>`__ to create your
+own development repository.
+
+After you successfully created your development repository, we will configure your existing `$HOME/src/custom`
+folder to be able to commit your work.
+
+
+.. code-block:: console
+
+    $ cd $HOME/src/custom
+    $ git init
+    $ git remote add training git@github.com:odoo/technical-training-sandbox.git
+    $ git remote add training-dev git@github.com:xyz-odoo/technical-training-sandbox.git #Don't forget to change xyz-odoo to your own GitHub account
+    $ git remote set-url --push training no_push #Avoid pushing directly into the main repository
+    $ git fetch --all --prune
+    $ git branch master training/master
+    $ git checkout master #Two new files will be downloaded README.md and .gitignore
+
+Note that if your work is already in a :ref:`Git and GitHub configured folder <howto/rdtraining/02_setup/development_repository>`
+the previous steps are not needed. This is going to be the case when you will start working in the
+'odoo' or 'enterprise' folders.
+
+
+Branch, Commit & Push
+---------------------
+
+Before creating the PR, a new working branch has to be created and used to commit the code. Afterwards
+the branch will be pushed to the development repository.
+
+Create your working branch:
+
+.. code-block:: console
+
+    $ git checkout -b master-my_first_branch-xyz
+
+Your branch name must follow the following name strucutre : <targetVersion>-<feature>-<trigram>
+
+Example: The branch master-sale-fixes-abc on odoo-dev/odoo is a branch containing fixes for the
+sales app in the odoo/odoo repository, to be deployed in master and done by ABC.
+
+Commit your code:
+
+
+.. code-block:: console
+
+    $ git add .
+    $ git commit
+
+
+**Everyone is reading your commit messages!**
+
+The commit message is very important, follow the :ref:`Developer guidelines <reference/guidelines/git>`.
+
+
+Push your new branch to your developement repository:
+
+.. code-block:: console
+
+    $ git push -u master-my_first_branch-xyz
```

Comment:
If you're on a checked-out branch (which is usually the case) `HEAD` is a shorter way to say this.

Also does this actually work without providing a remote name? Maybe because there's only one remote in this repo? Seems like a bad advice though.